### PR TITLE
Support for npm packages in parametric tests

### PR DESCRIPTION
**Pull request by hoolioh**
## Description

Include support for installing packaged tracers.

## Check list

Your PR is not ready to be reviewed? Please save it as a draft :pray:

- [x] Follows the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
- [x] CI is passing
- [ ] There is at least one approval from code owners

Yes to all? Feel free to merge it whenever you want :heart:

### CODE REVIEW

File: parametric/README.md [hunk 1]
```diff
@@ -138,6 +138,13 @@ To run the Python tests "locally" push your code to a branch and then specify ``
 PYTHON_DDTRACE_PACKAGE=git+https://github.com/Datadog/dd-trace-py@1.x ./run.sh ...
 ```
 
+#### NodeJS
+
+There is two ways for running the NodeJS tests with a custom tracer:
+- Place an NPM package in the folder ``apps/nodejs/npm`` and then set the enviornment variable ``NODEJS_DDTRACE_MODULE``
```
Comment (Kyle-Verhoog) [inline parametric/README.md:144]:
```suggestion
- Place the ddtrace NPM package in the folder ``apps/nodejs/npm`` and then set the enviornment variable ``NODEJS_DDTRACE_MODULE``
```
?

Comment (Kyle-Verhoog) [reply parametric/README.md:144]: can we maybe have a one liner command for how one can do this (given that the repo is cloned locally).

**Review by Kyle-Verhoog — APPROVED** [review]
Just a couple thoughts but otherwise lgtm! Thanks!

File: parametric/README.md (hunk 1 above)
Comment (hoolioh) [reply parametric/README.md:144]: Yes. I will add an example for each case. Thanks for your suggestions.

File: parametric/README.md [hunk 2]
```diff
@@ -141,9 +141,12 @@ PYTHON_DDTRACE_PACKAGE=git+https://github.com/Datadog/dd-trace-py@1.x ./run.sh .
 #### NodeJS
 
 There is two ways for running the NodeJS tests with a custom tracer:
-- Place an NPM package in the folder ``apps/nodejs/npm`` and then set the enviornment variable ``NODEJS_DDTRACE_MODULE``
-with the filename placed in the aforementioned folder, e.g., ``NODEJS_DDTRACE_MODULE="dd-trace-2.22.3.tgz"``
-- Set the environment variable to hold a commit in a remote branch, e.g., ``NODEJS_DDTRACE_MODULE=datadog/dd-trace-js#687cb813289e19bfcc884a2f9f634470cf138143``
+- Place the ddtrace NPM package in the folder ``apps/nodejs/npm`` and then set the enviornment variable ``NODEJS_DDTRACE_MODULE``
```
Comment (Kyle-Verhoog) [inline parametric/README.md:144]:
```suggestion
- Place the ddtrace NPM package in the folder ``apps/nodejs/npm`` and then set the environment variable ``NODEJS_DDTRACE_MODULE``
```

also, how does one get the npm package? or is this common knowledge for a node dev?

File: parametric/README.md [hunk 3]
```diff
@@ -141,9 +141,12 @@ PYTHON_DDTRACE_PACKAGE=git+https://github.com/Datadog/dd-trace-py@1.x ./run.sh .
...
+with the filename placed in the aforementioned folder. So after you've proceed with the installation and the ddtrace package
+properly placed you can run the tests with the following command:
```
Comment (Kyle-Verhoog) [inline parametric/README.md:146]:
```suggestion
```

File: parametric/README.md [hunk 4]
```diff
@@ -141,9 +141,12 @@ PYTHON_DDTRACE_PACKAGE=git+https://github.com/Datadog/dd-trace-py@1.x ./run.sh .
 #### NodeJS
 
 There is two ways for running the NodeJS tests with a custom tracer:
-- Place an NPM package in the folder ``apps/nodejs/npm`` and then set the enviornment variable ``NODEJS_DDTRACE_MODULE``
-with the filename placed in the aforementioned folder, e.g., ``NODEJS_DDTRACE_MODULE="dd-trace-2.22.3.tgz"``
-- Set the environment variable to hold a commit in a remote branch, e.g., ``NODEJS_DDTRACE_MODULE=datadog/dd-trace-js#687cb813289e19bfcc884a2f9f634470cf138143``
+- Place the ddtrace NPM package in the folder ``apps/nodejs/npm`` and then set the enviornment variable ``NODEJS_DDTRACE_MODULE``
+with the filename placed in the aforementioned folder. So after you've proceed with the installation and the ddtrace package
```
Comment (Kyle-Verhoog) [inline parametric/README.md:145]:
```suggestion
with the filename placed in the aforementioned folder. For example: 
```

**Review by Kyle-Verhoog — APPROVED** [review]
just a couple more nits but i'm ok to merge as-is

File: parametric/README.md (hunk 2 above)
Comment (hoolioh) [reply parametric/README.md:144]: Yep, it's pretty common for a node dev, in fact is simply running this command: `npm pack` in the project's folder.

**Review by Kyle-Verhoog — APPROVED** [review]

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**